src/3D/AllMeshes.cpp

```diff
@@ -0,0 +1,643 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "AllMeshes.h"
+
+namespace openblack
+{
+
+std::array<std::string, static_cast<uint16_t>(MeshId::_COUNT)> MeshNames = {
```

Comment:
In general +1 for moving this out of the header.
Could you make this a `constexpr` or `constinit` to make it threadsafe
Second, if there is any indication that this array can come from a resource file from the game, please make a note of it and write your code in function of the fact that in the future we want to load those strings from data files

Reply:
constexpr std::string support and constinit are both added in c++20